### ISSUE TITLE
note 6 at "contrast ratio" is unclear

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2268]
[note 6 at "contrast ratio"](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast#dfn-contrast-ratio) is unclear:

> WCAG conformance should be evaluated for color pairs specified in the content that an author would expect to appear adjacent in typical presentation. Authors need not consider unusual presentations, such as color changes made by the user agent, except where caused by authors' code. 

* What is a typical and an atypical presentation? 
* What are unusual presentations?
* What color changes makes the user agent?

Perhaps it should be made clear:
* If the user adjusts the colors in the operating system or in the user agent, the contrasts do not have to be respected (only if foreground and background colors are both either defined or not defined, see note 4).
* If the user adjusts the width of the screen or zooms in, the contrasts must be respected, even if the author did not plan for the text to then suddenly spill out of its container or for the text to suddenly be in a place on the background image where there is insufficient contrast

**Comment by JAWS-test** [comment]
Possibly note 6 also contradicts the new note 3 at 5.2.2 Full pages: 

> A full page includes each variation of the page that is automatically presented by the page for various screen sizes (e.g. variations in a responsive Web page). Each of these variations needs to conform (or needs to have a conforming alternate version) in order for the entire page to conform.
